src/helpers/chains.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  proxies,
+  baseURIs,
+  type TablelandNetworkConfig,
+} from "@tableland/evm/network.js";
+
+/**
+ * The set of supported chain names as used by the Tableland network.
+ */
+export type ChainName = keyof TablelandNetworkConfig;
+
+/**
+ * Chain information used to determine defaults for the set of supported chains.
+ */
+export interface ChainInfo {
+  chainName: ChainName;
+  chainId: number;
+  contractAddress: string;
+  baseUrl: string;
+  [key: string]: ChainInfo[keyof ChainInfo];
+}
+
+// We simply pull this automatically from @tableland/evm to avoid keeping track seperately here.
+const entries = Object.entries(proxies) as Array<[ChainName, string]>;
+const mapped = entries.map(([chainName, contractAddress]) => {
+  const uri = new URL(baseURIs[chainName]);
+  const baseUrl = `${uri.protocol}//${uri.host}/api/v1`;
+  const chainId = parseInt(uri.pathname.split("/")[2]);
+  const entry: [ChainName, any] = [
+    chainName,
+    { chainName, chainId, contractAddress, baseUrl },
+  ];
+  return entry;
+});
+
+/**
+ * The set of chains and their information as supported by the Tableland network.
+ */
+export const supportedChains = Object.fromEntries(mapped) as Record<
+  ChainName,
+  ChainInfo
+>;
+
+// Not exported
+const supportedChainsById = Object.fromEntries(
+  Object.values(supportedChains).map((v) => [v.chainId, v])
+);
+
+/**
+ * Get the default chain information for a given chain name.
```

Comment:
All the getters are here, and they'll take a number or chain name. The chain names should match with ethersjs once the PR in @tableland/evm is published and we pull that in here.